fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2652,6 +2652,10 @@ public class Config extends ConfigBase {
     @ConfField
     public static boolean checkpoint_after_check_compatibility = false;
 
+    // The max number of profiles that can be stored to storage.
+    @ConfField
+    public static int max_profile_on_storage = 500;
```

Comment:
rename to  max_spilled_profile_num = 500

Reply:
done